### PR TITLE
setup mock server

### PR DESCRIPTION
**Pull request by Alena-alena**

### CODE REVIEW

File: src/setupProxy.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = function(app) {
+    app.use(
+        '/api',
+        createProxyMiddleware({
+            target: 'http://localhost:9000/api',
```
Comment (haylen) [inline src/setupProxy.js]: better not to hardcode this value, but take it from the env var

File: src/services/BriefData.service.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const briefsUrl = `${process.env.REACT_APP_API_URL}/briefs`;
+const token = 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855760c11cd34f37e57d397dc2eeaefa68a';
```
Comment (haylen) [inline src/services/BriefData.service.js]: it is not considered a good practice to keep tokens hardcoded. They will now be kept on github in a commit history, so anyone in the future who has access to the repo can know your access token

File: src/services/BriefData.service.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
```
Comment (haylen) [inline src/services/BriefData.service.js:1]: stick to something one "" or ''

File: src/components/BriefList/BriefsList.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import {useEffect} from "react";
+import {getBriefList} from "../../services/BriefData.service";
+
+const BriefList = (props) => {
+  useEffect(() => {
+    getBriefList();
+  });
```
Comment (haylen) [inline src/components/BriefList/BriefsList.jsx]: overall when you use `useEffect` try to always provide the dependency array

Comment (haylen) [reply src/components/BriefList/BriefsList.jsx]: also you have different indentation in different files

File: src/services/BriefData.service.js (hunk 3 above)
Comment (Alena-alena) [reply src/services/BriefData.service.js:1]: This will be fixed after adding the linter

File: src/components/BriefList/BriefsList.jsx (hunk 4 above)
Comment (Alena-alena) [reply src/components/BriefList/BriefsList.jsx]: The indentation will be fixed after adding the linter

**Pull request merged**